aioesphomeapi/model.py

```diff
@@ -172,7 +173,7 @@ class CoverOperation(APIIntEnum):
     IS_CLOSING = 2
 
 
-@dataclass(frozen=True)
+@dataclass(frozen=True) 
```

Comment:
Oops, this was an accident. I'll amend it out

Reply:
```suggestion
@dataclass(frozen=True)
```